src/components/courseCard/courseCard1.js

```diff
@@ -53,11 +62,9 @@ const CourseCard = () => {
           }}
         />
         <div style={{ backgroundColor: "#000", color: "#fff", padding: "2px 8px" }}>
-          <Typography variant="h6">Chapter 1 : How to start with WD?</Typography>
+          <Typography variant="h5">Chapter 1 : How to start with WD?</Typography>
```

Comment:
PLease revert all the changes that you have done with variant. We are primarily developing this application for mobiles. If it looks too smaller on web we can do it with changing themes directory when building web apps.

Reply:
This doesn't looks good: 
![image](https://user-images.githubusercontent.com/48255244/111589053-5dc1a180-87ea-11eb-91ed-969b3f3a37c4.png)

Reply:
okay. so i'll make changes for mobile breakpoints then

Reply:
![image](https://user-images.githubusercontent.com/59202075/111595359-10e1c900-87f2-11eb-954b-77e021175b92.png)

the card view for mobile breakponts

Reply:
![image](https://user-images.githubusercontent.com/59202075/111595558-3ec70d80-87f2-11eb-9a1e-626cdae87f94.png)

cards view for mobile breakpoints

@Abhishek-kumar09 pls say if the above are fine or not